JsonRead/src/main/java/weather/Demo.java

```diff
@@ -0,0 +1,19 @@
+package weather;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.net.URL;
+
+public class Demo {
+    public static void main(String[] args) throws Exception {
+
+        String city = "Berlin,de";
+        String uRLString = "https://api.openweathermap.org/data/2.5/" +
```

Comment:
Think of how to make the application reusable and testable. Hints: define classes, which serve different purposes: one uses object mapper to read from the internet, another one deals with outputting wether in different formats, etc.

Reply:
more or less so?

Reply:
URLString is a property of a class, which is responsible for getting data from the Internet. city on the other hand, is a parameter, which denotes the city I need to know the weather in. As user, I do not need (and should not) know which server/URL is used to get the data from. As well as your api id should not be visible to clients.